AUTHORS

```diff
@@ -868,5 +867,128 @@ Super Zheng <superzheng@tencent.com>
 Rafał Pocztarski <r.pocztarski@gmail.com>
 Michael Ruddy <mruddybtc@gmail.com>
 Andy Bettisworth <andy.bettisworth@accreu.com>
+Jörg Krause <joerg.krause@embedded.rocks>
+Alexander Martin <alex@suitupalex.com>
+Prince J Wesley <princejohnwesley@gmail.com>
+janriemer <janriemer@users.noreply.github.com>
+Arthur Gautier <superbaloo+registrations.github@superbaloo.net>
+Martin von Gagern <Martin.vGagern@gmx.net>
+Hideki Yamamura <hideki.yamamura@gmail.com>
+Lenny Markus <lmarkus@paypal.com>
+Nelson Pecora <nelson@yoshokatana.com>
+ronkorving <ron@ronkorving.nl>
```

Comment:
duplicate... @ronkorving, your old entry shows a different email address, do you want it updated to this one?